packages/notification-center/src/components/popover-notification-center/components/Popover.tsx

```diff
@@ -25,6 +33,11 @@ export function Popover({ children, bell, theme, offset = 0, position = 'bottom-
   };
   const { markFetchedNotificationsAsSeen } = useNotifications();
 
+  useEffect(() => {
+    if (isOpened) {
+      setIsVisible(true);
+    }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Should we change this to allow `isOpened` to control either state? Seems like that's what users would want
‏

Reply:
Yes, and I would say that this will require more changes in the code because currently, the NC popover closes when clicking outside, and if we want to provide the option to control the open state then that behavior should not impact that open decision.

Reply:
Additionally, we should mark notifications as seen when it's getting closed.